full_stack_tests/script.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -x
-rm -f test_output.txt
-cd ../build
```

Comment:
Wondering if this is the problem. Might need to move to the `build` directory before we run tests. Maybe they work on my local machine because I always navigate there manually first.